pom.xml

```diff
@@ -1365,6 +1365,32 @@
               <skip>${code.coverage.disabled}</skip>
             </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>gwt-maven-plugin</artifactId>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>sonarcloud-analysis</id>
+      <properties>
+        <skipITs>true</skipITs>
+        <sonar.skip>true</sonar.skip>
```

Comment:
Wouldn't it make sense to have the sonar.skip property in general config (project > properties), not in a profile, and just override value true/false in particular modules as needed?

Reply:
Thank you for pointing to this, actually I will remove `sonarcloud-analysis` profile config completely, for now.

I know we use it for the rest of kiegroup organization, however I was not able to get it working for kogito-editors-java, I would like to proceed with command generated by soanrcloud:
```
org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
```